### PR TITLE
Logging component backup fix

### LINKED ISSUE #2707: [BUG] Logging component backup fails
**Describe the bug**
I'm not able to backup the logging component.
It fails when checking cluster health:
```
---
- name: Set helper facts
  set_fact:
    elasticsearch_endpoint: >-
      https://{{ ansible_default_ipv4.address }}:9200
    snapshot_name: >-
      {{ ansible_date_time.iso8601_basic_short | replace('T','-') }}
  vars:
    uri_template: &uri
      client_cert: /etc/elasticsearch/kirk.pem
      client_key: /etc/elasticsearch/kirk-key.pem
      validate_certs: false
      body_format: json

- debug: var=snapshot_name

- name: Check cluster health
  uri:
    <<: *uri
    url: "{{ elasticsearch_endpoint }}/_cluster/health"
    method: GET
  register: uri_response
  until: uri_response is success
  retries: 12
  delay: 5
```
```
14:29:27 INFO cli.engine.ansible.AnsibleCommand - TASK [backup : Check cluster health] *******************************************
14:29:28 ERROR cli.engine.ansible.AnsibleCommand - FAILED - RETRYING: Check cluster health (12 retries left).
14:29:34 ERROR cli.engine.ansible.AnsibleCommand - FAILED - RETRYING: Check cluster health (11 retries left).
14:29:39 ERROR cli.engine.ansible.AnsibleCommand - FAILED - RETRYING: Check cluster health (10 retries left).
14:29:45 ERROR cli.engine.ansible.AnsibleCommand - FAILED - RETRYING: Check cluster health (9 retries left).
14:29:51 ERROR cli.engine.ansible.AnsibleCommand - FAILED - RETRYING: Check cluster health (8 retries left).
14:29:57 ERROR cli.engine.ansible.AnsibleCommand - FAILED - RETRYING: Check cluster health (7 retries left).
14:30:03 ERROR cli.engine.ansible.AnsibleCommand - FAILED - RETRYING: Check cluster health (6 retries left).
14:30:08 ERROR cli.engine.ansible.AnsibleCommand - FAILED - RETRYING: Check cluster health (5 retries left).
14:30:14 ERROR cli.engine.ansible.AnsibleCommand - FAILED - RETRYING: Check cluster health (4 retries left).
14:30:20 ERROR cli.engine.ansible.AnsibleCommand - FAILED - RETRYING: Check cluster health (3 retries left).
14:30:26 ERROR cli.engine.ansible.AnsibleCommand - FAILED - RETRYING: Check cluster health (2 retries left).
14:30:31 ERROR cli.engine.ansible.AnsibleCommand - FAILED - RETRYING: Check cluster health (1 retries left).
14:30:37 ERROR cli.engine.ansible.AnsibleCommand - An exception occurred during task execution. To see the full traceback, use -vvv. The error was: IOError: [Errno 2] No such file or directory
14:30:37 ERROR cli.engine.ansible.AnsibleCommand - fatal: [qa-azuretestfull-logging-vm-0]: FAILED! => {"attempts": 12, "changed": false, "elapsed": 0, "msg": "Status code was -1 and not [200]: An unknown error occurred: [Errno 2] No such file or directory", "redirected": false, "status": -1, "url": "https://10.1.3.4:9200/_cluster/health"}
```

When executed directly on the logging vm:
```
root@ec2-x-x-x-x:/etc/elasticsearch# curl -k --cert kirk.pem --key kirk-key.pem https://10.1.3.109:9200/_cluster/health
curl: (58) could not load PEM client certificate, OpenSSL error error:02001002:system library:fopen:No such file or directory, (no key found, wrong pass phrase, or wrong file format?)
```

This is due to removal of the demo certificates and now `epiphany-admin-key.pem` and `epiphany-admin.pem` should be used instead of `kirk-key.pem` and `kirk.pem` which don't exist.
```
root@ec2-x-x-x-x:/etc/elasticsearch# ls -la /etc/elasticsearch/
total 84
drwxr-s---   5 root elasticsearch  4096 Oct 15 07:37 .
drwxr-xr-x 100 root root           4096 Oct 15 07:40 ..
-rw-r--r--   1 root elasticsearch    76 Oct 15 07:36 .elasticsearch.keystore.initial_md5sum
drwxr-x---   2 root elasticsearch  4096 Oct 15 07:37 csr
-rw-rw----   1 root elasticsearch   199 Oct 15 07:36 elasticsearch.keystore
-rw-rw----   1 root elasticsearch  4621 Oct 15 07:37 elasticsearch.yml
-rw-rw----   1 root elasticsearch  4547 Oct 15 07:36 elasticsearch.yml.20433.2021-10-15@07:37:23~
-rw-------   1 root elasticsearch  1708 Oct 15 07:37 epiphany-admin-key.pem
-rw-r-----   1 root elasticsearch  1285 Oct 15 07:37 epiphany-admin.pem
-rw-r-----   1 root elasticsearch  1704 Oct 15 07:37 epiphany-node-ec2-x-x-x-x.eu-west-1.compute.amazonaws.com-key.pem
-rw-r--r--   1 root elasticsearch  1464 Oct 15 07:37 epiphany-node-ec2-x-x-x-x.eu-west-1.compute.amazonaws.com.pem
-rw-r-----   1 root elasticsearch  1233 Oct 15 07:37 epiphany-root-ca.pem
-rw-rw----   1 root elasticsearch  2600 Oct 15 07:36 jvm.options
-rw-rw----   1 root elasticsearch  2581 Oct 15 07:36 jvm.options.19923.2021-10-15@07:37:00~
drwxr-s---   2 root elasticsearch  4096 Jan 13  2021 jvm.options.d
-rw-rw----   1 root elasticsearch 11021 Jan 13  2021 log4j2.properties
drwxr-x---   2 root elasticsearch  4096 Oct 15 07:37 private

```
```
root@ec2-x-x-x-x:/etc/elasticsearch# curl -s -k --cert epiphany-admin.pem --key epiphany-admin-key.pem https://10.1.3.109:9200/_cluster/health | jq
{
  "cluster_name": "EpiphanyElastic",
  "status": "yellow",
  "timed_out": false,
  "number_of_nodes": 1,
  "number_of_data_nodes": 1,
  "active_primary_shards": 6,
  "active_shards": 6,
  "relocating_shards": 0,
  "initializing_shards": 0,
  "unassigned_shards": 4,
  "delayed_unassigned_shards": 0,
  "number_of_pending_tasks": 0,
  "number_of_in_flight_fetch": 0,
  "task_max_waiting_in_queue_millis": 0,
  "active_shards_percent_as_number": 60
}
```

Additionally, at this point, we only "support" single node backup. Consider a multi-node cluster backup:
https://github.com/epiphany-platform/epiphany/issues/1416
https://github.com/epiphany-platform/epiphany/issues/1417

**How to reproduce**
1. Deploy a cluster with 1 logging vm
2. Run backup command: `epicli backup -b /shared/build/testbackup -f /shared/build/backup-config.yml`

**Expected behavior**
Backup should be successful

**Config files**
Backup config:
```
kind: configuration/backup
title: Backup Config
name: default
provider: azure
specification:
  components:
    load_balancer:
      enabled: false
    logging:
      enabled: true
    monitoring:
      enabled: false
    postgresql:
      enabled: false
    rabbitmq:
      enabled: false
    kubernetes:
      enabled: false
```
Recovery config:
```
kind: configuration/recovery
title: Recovery Config
name: default
provider: azure
specification:
  components:
    load_balancer:
      enabled: false
      snapshot_name: latest
    logging:
      enabled: true
      snapshot_name: latest
    monitoring:
      enabled: false
      snapshot_name: latest
    postgresql:
      enabled: false
      snapshot_name: latest
    rabbitmq:
      enabled: false
      snapshot_name: latest
```

**Environment**
- Cloud provider: [all]
- OS: [all]

**epicli version**: [`epicli --version`]
1.3.0dev
Previous versions are probably also affected.

**Additional context**

---

**DoD checklist**

* [x] Changelog updated (if affected version was released)
* [x] COMPONENTS.md updated / doesn't need to be updated
* [x] Automated tests passed (QA pipelines)
  * [ ] apply
  * [ ] upgrade
* [x] Case covered by automated test (if possible)
* [ ] Idempotency tested
* [x] Documentation updated / doesn't need to be updated
* [x] All conversations in PR resolved
* [x] Backport tasks created / doesn't need to be backported

### PR DESCRIPTION
**Pull request by plirglo**
issue: #2707

### CODE REVIEW

**Review by rafzei — DISMISSED** [review]
LGTM

**Comment by przemyslavic** [comment]
Filebeat service is not running on logging vm after recovery.
```
2021-11-18T11:48:46.5070646Z [38;21m11:48:46 INFO cli.engine.ansible.AnsibleCommand - TASK [recovery : Start all filebeat instances] *********************************[0m
2021-11-18T11:48:47.8594913Z [38;21m11:48:47 INFO cli.engine.ansible.AnsibleCommand - changed: [ci-recazurrhelflannel-logging-vm-0 -> ci-recazurrhelflannel-logging-vm-0] => (item=ci-recazurrhelflannel-logging-vm-0)[0m
2021-11-18T11:48:49.1618454Z [38;21m11:48:49 INFO cli.engine.ansible.AnsibleCommand - changed: [ci-recazurrhelflannel-logging-vm-0 -> ci-recazurrhelflannel-repository-vm-0] => (item=ci-recazurrhelflannel-repository-vm-0)[0m
```
```
[operations@ci-recazurrhelcanal-logging-vm-0 ~]$ systemctl status filebeat
● filebeat.service - Filebeat sends log files to Logstash or directly to Elasticsearch.
   Loaded: loaded (/usr/lib/systemd/system/filebeat.service; enabled; vendor preset: disabled)
   Active: failed (Result: start-limit) since Thu 2021-11-18 11:48:12 UTC; 13min ago
     Docs: https://www.elastic.co/products/beats/filebeat
  Process: 27058 ExecStart=/usr/share/filebeat/bin/filebeat --environment systemd $BEAT_LOG_OPTS $BEAT_CONFIG_OPTS $BEAT_PATH_OPTS (code=exited, status=1/FAILURE)
 Main PID: 27058 (code=exited, status=1/FAILURE)

Nov 18 11:48:12 ci-recazurrhelcanal-logging-vm-0 filebeat[27058]: 2021-11-18T11:48:12.043Z        ERROR        instance/beat.go:951        Exiting: error...
Nov 18 11:48:12 ci-recazurrhelcanal-logging-vm-0 systemd[1]: Unit filebeat.service entered failed state.
Nov 18 11:48:12 ci-recazurrhelcanal-logging-vm-0 filebeat[27058]: Exiting: error connecting to Kibana: fail to get the Kibana version: HTTP GET request t...
Nov 18 11:48:12 ci-recazurrhelcanal-logging-vm-0 systemd[1]: filebeat.service failed.
Nov 18 11:48:12 ci-recazurrhelcanal-logging-vm-0 systemd[1]: filebeat.service holdoff time over, scheduling restart.
Nov 18 11:48:12 ci-recazurrhelcanal-logging-vm-0 systemd[1]: Stopped Filebeat sends log files to Logstash or directly to Elasticsearch..
Nov 18 11:48:12 ci-recazurrhelcanal-logging-vm-0 systemd[1]: start request repeated too quickly for filebeat.service
Nov 18 11:48:12 ci-recazurrhelcanal-logging-vm-0 systemd[1]: Failed to start Filebeat sends log files to Logstash or directly to Elasticsearch..
Nov 18 11:48:12 ci-recazurrhelcanal-logging-vm-0 systemd[1]: Unit filebeat.service entered failed state.
Nov 18 11:48:12 ci-recazurrhelcanal-logging-vm-0 systemd[1]: filebeat.service failed.
Hint: Some lines were ellipsized, use -l to show in full.
```
It is up after executing `sudo systemctl start filebeat` manually.
```
[operations@ci-recazurrhelcanal-logging-vm-0 ~]$ systemctl status filebeat
● filebeat.service - Filebeat sends log files to Logstash or directly to Elasticsearch.
   Loaded: loaded (/usr/lib/systemd/system/filebeat.service; enabled; vendor preset: disabled)
   Active: active (running) since Thu 2021-11-18 12:02:30 UTC; 1s ago
     Docs: https://www.elastic.co/products/beats/filebeat
 Main PID: 6954 (filebeat)
   CGroup: /system.slice/filebeat.service
           └─6954 /usr/share/filebeat/bin/filebeat --environment systemd -c /etc/filebeat/filebeat.yml --path.home /usr/share/filebeat --path.config /etc...

Nov 18 12:02:30 ci-recazurrhelcanal-logging-vm-0 filebeat[6954]: 2021-11-18T12:02:30.816Z        INFO        [beat]        instance/beat.go:992        Ho...
Nov 18 12:02:30 ci-recazurrhelcanal-logging-vm-0 filebeat[6954]: 2021-11-18T12:02:30.816Z        INFO        [beat]        instance/beat.go:1021        P...
Nov 18 12:02:30 ci-recazurrhelcanal-logging-vm-0 filebeat[6954]: 2021-11-18T12:02:30.816Z        INFO        instance/beat.go:299        Setup Bea...: 7.9.2
Nov 18 12:02:30 ci-recazurrhelcanal-logging-vm-0 filebeat[6954]: 2021-11-18T12:02:30.816Z        INFO        eslegclient/connection.go:99        e...-0:9200
Nov 18 12:02:30 ci-recazurrhelcanal-logging-vm-0 filebeat[6954]: 2021-11-18T12:02:30.816Z        WARN        [tls]        tlscommon/tls_config.go:...sabled.
Nov 18 12:02:30 ci-recazurrhelcanal-logging-vm-0 filebeat[6954]: 2021-11-18T12:02:30.817Z        INFO        [publisher]        pipeline/module.go...ng-vm-0
Nov 18 12:02:30 ci-recazurrhelcanal-logging-vm-0 filebeat[6954]: 2021-11-18T12:02:30.818Z        INFO        [monitoring]        log/log.go:118   ...ery 30s
Nov 18 12:02:30 ci-recazurrhelcanal-logging-vm-0 filebeat[6954]: 2021-11-18T12:02:30.818Z        INFO        kibana/client.go:119        Kibana ur...-0:5601
Nov 18 12:02:30 ci-recazurrhelcanal-logging-vm-0 filebeat[6954]: 2021-11-18T12:02:30.841Z        INFO        [add_cloud_metadata]        add_cloud_metada...
Nov 18 12:02:30 ci-recazurrhelcanal-logging-vm-0 filebeat[6954]: 2021-11-18T12:02:30.965Z        INFO        kibana/client.go:119        Kibana ur...-0:5601
Hint: Some lines were ellipsized, use -l to show in full.
```

From what I can see it's not strictly related to this PR, but I think it needs to be fixed here.

File: docs/home/howto/BACKUP.md [hunk 1]
```diff
@@ -12,12 +12,12 @@ Epiphany provides solution to create full or partial backup and restore for some
 - [Kubernetes (only backup)](#kubernetes)
 
 Backup is created directly on the machine where component is running, and it is moved to the ``repository`` host via
-rsync. On the ``repository`` host backup files are stored in location ``/epibackup/mounted`` mounted on a local
+rsync. On the ``repository`` host backup files by default are stored in location ``/epibackup/mounted`` mounted on a local
```
Comment (to-bar) [inline docs/home/howto/BACKUP.md]: Is this location configurable?
I see it in ansible/playbooks/roles/backup/defaults/main.yml and not in schema/common/defaults/configuration/backup.yml
If not, I would not add `by default` 😉

Comment (plirglo) [reply docs/home/howto/BACKUP.md]: Nope, it's not configurable.

**Comment by plirglo** [comment]
@przemyslavic changes have been done, please re-test
image

**Comment by przemyslavic** [comment]
Fixed
```
2021-11-22T11:55:29.7076408Z 11:55:29 INFO cli.engine.ansible.AnsibleCommand - TASK [recovery : Start all filebeat instances] *********************************
2021-11-22T11:55:30.4868419Z 11:55:30 ERROR cli.engine.ansible.AnsibleCommand - FAILED - RETRYING: Start all filebeat instances (30 retries left).
2021-11-22T11:55:40.9753993Z 11:55:40 ERROR cli.engine.ansible.AnsibleCommand - FAILED - RETRYING: Start all filebeat instances (29 retries left).
2021-11-22T11:55:51.6192730Z 11:55:51 INFO cli.engine.ansible.AnsibleCommand - ok: [ci-recazurcentflannel-logging-vm-0 -> ci-recazurcentflannel-logging-vm-0] => (item=ci-recazurcentflannel-logging-vm-0)
2021-11-22T11:55:52.3255512Z 11:55:52 ERROR cli.engine.ansible.AnsibleCommand - FAILED - RETRYING: Start all filebeat instances (30 retries left).
2021-11-22T11:56:02.7544268Z 11:56:02 INFO cli.engine.ansible.AnsibleCommand - ok: [ci-recazurcentflannel-logging-vm-0 -> ci-recazurcentflannel-monitoring-vm-0] => (item=ci-recazurcentflannel-monitoring-vm-0)
2021-11-22T11:56:03.5741962Z 11:56:03 ERROR cli.engine.ansible.AnsibleCommand - FAILED - RETRYING: Start all filebeat instances (30 retries left).
2021-11-22T11:56:14.0998950Z 11:56:14 INFO cli.engine.ansible.AnsibleCommand - ok: [ci-recazurcentflannel-logging-vm-0 -> ci-recazurcentflannel-load-balancer-vm-0] => (item=ci-recazurcentflannel-load-balancer-vm-0)
2021-11-22T11:56:14.8851694Z 11:56:14 ERROR cli.engine.ansible.AnsibleCommand - FAILED - RETRYING: Start all filebeat instances (30 retries left).
2021-11-22T11:56:25.3477078Z 11:56:25 INFO cli.engine.ansible.AnsibleCommand - ok: [ci-recazurcentflannel-logging-vm-0 -> ci-recazurcentflannel-repository-vm-0] => (item=ci-recazurcentflannel-repository-vm-0)
```

**Review by przemyslavic — APPROVED** [review]

**Review by seriva — APPROVED** [review]

**Pull request merged**